src/Microsoft.Azure.WebJobs.Host/Loggers/ValueWatcher.cs

```diff
@@ -15,120 +15,4 @@
 
 namespace Microsoft.Azure.WebJobs.Host.Executors
 {
-    internal class ValueWatcher
```

Comment:
It's appears to be dead code and not used. Dashboard builds fine without it.  Only thing that's used is the static call to AddLogs, which got moved to parameter helper.  Do you see a usage somewhere I'm missing?

---
In reply to: [125713772](https://github.com/Azure/azure-webjobs-sdk/pull/1209#discussion_r125713772) [](ancestors = 125713772)